src/SFA.DAS.ApplyService.Web/SFA.DAS.ApplyService.Web.csproj

```diff
@@ -37,6 +37,7 @@
   <ItemGroup>
     <ProjectReference Include="..\SFA.DAS.ApplyService.Application\SFA.DAS.ApplyService.Application.csproj" />
     <ProjectReference Include="..\SFA.DAS.ApplyService.Configuration\SFA.DAS.ApplyService.Configuration.csproj" />
+    <ProjectReference Include="..\SFA.DAS.ApplyService.Data\SFA.DAS.ApplyService.Data.csproj" />
```

Comment:
No hard-linking to the Data project! Remove and use appropriate API calls & Handler please

Reply:
It's done